src/actions/websocket.js

```diff
@@ -131,6 +131,11 @@ async function handleReconnect(dispatch, getState) {
     getLicenseConfig()(dispatch, getState);
     getClientConfig()(dispatch, getState);
     getMyPreferences()(dispatch, getState);
+    getMe()(dispatch, getState);
```

Comment:
why do you need to get the current user? is not enough getting the updated status?